documentation/src/main/asciidoc/mapstruct-reference-guide.asciidoc

```diff
@@ -999,7 +999,11 @@ When working with JAXB, e.g. when converting a `String` to a corresponding `JAXB
 [[selection-based-on-qualifiers]]
 === Mapping method selection based on qualifiers
 
-In many occasions one requires mapping methods with the same method signature (appart from the name) that have different behavior. MapStruct has a handy mechanism to deal with such situations: `@Qualifier`. A ‘qualifier’ is a custom annotation that the user can write, ‘stick onto’ a mapping method which is included as used mapper, and can be referred to in a bean property mapping, iterable mapping or map mapping.  Multiple qualifiers can be ‘stuck onto’ a method and mapping.
+In many occasions one requires mapping methods with the same method signature (appart from the name) that have different behavior.
+MapStruct has a handy mechanism to deal with such situations: `@Qualifier` (`org.mapstruct.Qualifier`).
```

Comment:
In this changes, only the additiona of org.mapstruct.Qualifier is new. The rest is just to split the line into multiple